### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by mrnadaara**

### CODE REVIEW

**Comment by rna** [comment]
Changed the base branch to development branch

File: bin/main.rb [hunk 1]
```diff
@@ -1,4 +1,99 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-puts 'Hello Worlds!'
+$board
```
Comment (maelfosso) [inline bin/main.rb]: Not Allowed in this project.

File: bin/main.rb [hunk 2]
```diff
@@ -1,4 +1,99 @@
...
+def display_board
+  puts '----------------'
+  $board.each do |sub|
+    sub.each do |c|
+      print "| #{c} |"
+    end
+    puts ' '
+  end
+  puts '----------------'
+end
```
Comment (maelfosso) [inline bin/main.rb]: you can have `display_board(board)`

File: bin/main.rb [hunk 3]
```diff
@@ -1,4 +1,99 @@
...
+  $board = [%w[1 2 3], %w[4 5 6], %w[7 8 9]]
+  active_player = 1
+  # result = false
+
+  counter = 0
+  endgame = false
+
+  while counter < 9
+
+    puts "Player-#{active_player}: Please enter box number:-"
```
Comment (maelfosso) [inline bin/main.rb]: Don't you think about displaying the board to the user at the beginning ? Before asking him to enter a number ?

**Pull request merged**